tests/ci/codebuild/release/release.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+env:
+  variables:
+    GO_TAG: go1.18.3
+phases:
+  # TODO: Everything in the install section should be moved into a prebuilt image
+  install:
+    runtime-versions:
+      java: corretto11
+    commands:
+      - yum -y update
+      - yum -y group install 'Development Tools'
+      - curl -L -O https://downloads.sourceforge.net/ltp/lcov-1.14-1.noarch.rpm
+      - yum -y install cmake3
+      - yum -y install gsl-devel wget
+      - mkdir go_tmp
+      - cd go_tmp
+      - wget https://dl.google.com/go/$GO_TAG.linux-amd64.tar.gz
+      - tar -xf $GO_TAG.linux-amd64.tar.gz
+      - mv go ../go_root
+      - cd ..
+  build:
+    commands:
+      # TODO: Move this to the docker image also
+      - export GOROOT=$(pwd)/go_root
+      - export GO111MODULE=on
+      - export PATH=$GOROOT/bin:$PATH
+      # TODO: Can we move the recursive pull logic upstream?
+      - git submodule update --init --recursive
+      - pwd
+      - ls
+      # TODO: Uncomment next line to actually release
+      # - ./gradlew -DcmakeBin=cmake3 -DFIPS=${FIPS} -DstagingProperties=true releaseSonatypeStagingRepository
+      - ./gradlew -DcmakeBin=cmake3 -DFIPS=${FIPS} -DstagingProperties=true tasks
```

Comment:
nit: could you please add a comment explaining the purpose of this no-op `tasks` invocation?

Reply:
Sure.